builtInFunctions/keyValueStorage_test.go

```diff
@@ -85,23 +85,23 @@ func TestSaveKeyValue_ProcessBuiltinFunction(t *testing.T) {
 	value := []byte("value")
 	vmInput.Arguments = [][]byte{key, value}
 
-	_, err = skv.ProcessBuiltinFunction(nil, nil, vmInput)
+	_, err = skv.ProcessBuiltinFunction(acc, nil, vmInput)
 	require.Equal(t, ErrNilSCDestAccount, err)
 
-	_, err = skv.ProcessBuiltinFunction(acc, nil, vmInput)
+	_, err = skv.ProcessBuiltinFunction(acc, acc, vmInput)
```

Comment:
`(nil, acc)` in the entire file where the destination isn't nil?
the sender is ignored so maybe reflect this in the tests as well

Reply:
blockchain hook will never provide a nil account for sender. It is against how the protocol was implemented.

Reply:
will add one more test for nil sender